src/phoenix/experimental/evals/models/vertex.py

```diff
@@ -50,6 +50,9 @@ def __post_init__(self) -> None:
             max_retries=self.max_retries,
         )
 
+    def reload_client(self) -> None:
+        self._init_client()
+
     def _init_client(self) -> None:
         try:
             from google.api_core import exceptions  # type:ignore
```

Comment:
Long-term, maybe some of this logic should move into the base class if we are finding that all the clients need to be reinitialized.

Reply:
I think the way we initialize every SDK is a little different so I don't want to overgeneralize on how we might need to deal with resetting per event loop